src/constants.js

```diff
@@ -3,9 +3,9 @@ import os from 'os';
 
 import {version} from '../package.json';
 
-export const DEFAULT_SAUCE_CONNECT_VERSION = '5.2.2';
+export const DEFAULT_SAUCE_CONNECT_VERSION = '5';
```

Comment:
Does it a container or how will it know latest version?

Reply:
It makes requests to saucelabs api using this parameter: https://api.us-west-1.saucelabs.com/rest/v1/public/tunnels/sauce-connect/download?arch=arm64&os=macos&version=5